lib/node/Status.js

```diff
@@ -81,7 +81,7 @@ Status.childChanged = function(node, childStatus, childArgIndex=null) {
     return substeps;
   }
 
-  if (Type.isParenthesis(node)) {
+  if (node.type === 'Parentheses') {
```

Comment:
@aliang8  is adding `isParens` to `math-nodes` in https://github.com/semantic-math/math-nodes/pull/4/files.